### DISCUSSION TITLE
Bulk Invoices Sending

### DISCUSSION THREAD

**Discussion by jianweng11** [phax/phase4#283] [Q&A]
Dear Philip,

Is it possible to support bulk sending? The current standalone project seems to only support single invoice. When attempting to send multiple invoices, an invalid XML issue is encountered.

```
<Invoice xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2" xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
...
</Invoice>
<Invoice xmlns="urn:oasis:names:specification:ubl:schema:xsd:Invoice-2" xmlns:cac="urn:oasis:names:specification:ubl:schema:xsd:CommonAggregateComponents-2" xmlns:cbc="urn:oasis:names:specification:ubl:schema:xsd:CommonBasicComponents-2">
...
</Invoice>
```

Thanks.

**Comment by phax** [comment]
Hi @jianweng11
In the context of Peppol, each Invoice needs to be transmitted separately. Support for bulk invoices is not contained.
In any other context, please take into consideration, that XML must have a single root element, so it should be something like
```xml
<InvoiceList>
  <Invoice ...>
     ---
  </Invoice>
  <Invoice ...>
     ---
  </Invoice>
</InvoiceList>
```

**Comment by jianweng11** [comment]
Dear @phax,

Appreciate your prompt response, and many thanks. Singapore has a requirement to send bulk invoices as outlined below. Therefore, we are checking to confirm whether this can be supported.

[Sample Payload](https://drive.google.com/file/d/1-vuYs0EAQpm9RF4z2Zgmv4pgZaNg2NN8/view?usp=sharing)
[AP Testing](https://go.gov.sg/c5-testing-ap)

Thanks.

**Comment by phax** [comment]
This payload is not valid, because the SBDH XSD allows only for a sinlge payload document. I will reach out to the SG PA to tackle that issue.

**Comment by phax** [comment]
Okay, so the use case is only for tax reporting, and not for regular Peppol exchanges.
Will let you know about the outcome

**Comment by jianweng11** [comment]
Noted with thanks 🙏🏻
